packages/alea-frontend/pages/index.tsx

```diff
@@ -434,3 +422,16 @@ const StudentHomePage: NextPage = () => {
 };
 
 export default StudentHomePage;
+
+export async function getStaticProps() {
+  const mmtUrl = 'https://stexmmt.mathhub.info';
+  const courses = await getCourseInfo(mmtUrl);
+  const filteredKeys = Object.keys(courses).filter((key) =>
+    FEATURED_COURSES.includes(courses[key].courseId)
+  );
+  const filteredCourses = filteredKeys.map((key) => courses[key]);
+  return {
+    props: { filteredCourses },
+    revalidate: 10,
```

Comment:
10 seconds was only for testing. This will cause way too many revalidations. Set some reasonable value like 1hr